### PR TITLE
KAFKA-16041: Replace Afterburn module with Blackbird

### PR DESCRIPTION
**Pull request by fonsdant**

### CODE REVIEW

File: gradle/dependencies.gradle [hunk 1]
```diff
@@ -161,7 +161,7 @@ libs += [
   jacksonDataformatCsv: "com.fasterxml.jackson.dataformat:jackson-dataformat-csv:$versions.jackson",
   jacksonModuleScala: "com.fasterxml.jackson.module:jackson-module-scala_$versions.baseScala:$versions.jackson",
   jacksonJDK8Datatypes: "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:$versions.jackson",
-  jacksonAfterburner: "com.fasterxml.jackson.module:jackson-module-afterburner:$versions.jackson",
+  jacksonBlackburn: "com.fasterxml.jackson.module:jackson-module-blackbird:$versions.jackson",
```
Comment (mimaison) [inline gradle/dependencies.gradle]: Can we call it `jacksonBlackbird` instead of `jacksonBlackburn`?

**Review by mimaison — COMMENTED** [review]
Thanks for the PR! I left a comment.

File: gradle/dependencies.gradle (hunk 1 above)
Comment (fonsdant) [reply gradle/dependencies.gradle]: Oh, It was a typo! Thanks, @mimaison! I'm gonna push a fix.

**Review by mimaison — APPROVED** [review]
LGTM, thanks for the PR

**Comment by fonsdant** [comment]
Thanks, Mickael! :)

**Pull request merged**